eclass/distutils-r1.eclass

```diff
@@ -1008,35 +996,22 @@ distutils_pep517_install() {
 	[[ ${#} -eq 1 ]] || die "${FUNCNAME} takes exactly one argument: root"
 
 	local root=${1}
-	local -x WHEEL_BUILD_DIR=${BUILD_DIR}/wheel
-	mkdir -p "${WHEEL_BUILD_DIR}" || die
+	local wheel_dir=${BUILD_DIR}/wheel
+	mkdir -p "${wheel_dir}" || die
 
 	local build_backend=$(_distutils-r1_get_backend)
 	einfo "  Building the wheel for ${PWD#${WORKDIR}/} via ${build_backend}"
 	local wheel=$(
-		"${EPYTHON}" - 3>&1 >&2 <<-EOF || die "Wheel build failed"
-			import ${build_backend%:*}
-			import os
-			print(${build_backend/:/.}.build_wheel(os.environ['WHEEL_BUILD_DIR']),
-				file=os.fdopen(3, 'w'))
-		EOF
+		gpep517 build-wheel --backend "${build_backend}" \
+				--output-fd 3 --wheel-dir "${wheel_dir}" 3>&1 >&2 ||
+			die "Wheel build failed"
 	)
 	[[ -n ${wheel} ]] || die "No wheel name returned"
 
 	einfo "  Installing the wheel to ${root}"
-	# NB: --compile-bytecode does not produce the correct paths,
-	# and python_optimize doesn't handle being called outside D,
-	# so we just defer compiling until the final merge
-	# NB: we override sys.prefix & sys.exec_prefix because otherwise
-	# installer would use virtualenv's prefix
```

Comment:
Just a question: is this comment irrelevant with new nicer code?

Reply:
The first one is still kinda relevant but I've pretty much assumed it's status quo to be doing compiling later. The second one is not.